tensorboard/plugins/graph/README.md

```diff
@@ -438,7 +438,11 @@ Note: building a Hierarchy modifies the SlimGraph's OpNodes in place!
 
 ### Rendering the view
 
-TODO
+#### **XLA Cluster Visualization**
```

Comment:
nit: can probably drop the `**` right? I think headers are usually bolded already?

Reply:
It didn't look very bold on my visual studio preview, maybe it's just the VS I'm using. Dropping ** for better format.